src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -63,7 +64,20 @@
 							<ClockOutline v-else :size="16" />
 						</template>
 						{{ messageDateTime }}
-					</NcActionButton>
+					</NcActionText>
+					<!-- Edited message timestamp -->
+					<NcActionButtonGroup v-if="messageObject.lastEditTimestamp">
+						<NcActionText>
+							<template #icon>
+								<ClockEditOutline :size="16" />
+							</template>
+							{{ messageObject.lastEditActorDisplayName }}
+						</NcActionText>
+						<NcActionText>
+							{{ editedDateTime }}
+						</NcActionText>
```

Comment:
```suggestion
						<NcActionText :name="messageObject.lastEditActorDisplayName">
							<template #icon>
								<ClockEditOutline :size="16" />
							{{ editedDateTime }}
						</NcActionText>
```
I guess, that was Joas' idea 
![image](https://github.com/nextcloud/spreed/assets/93392545/8c1faecf-5607-4c9a-b889-45702a150ef7)

Reply:
Bold is a bit peculiar to the message buttons style :/